### PR TITLE
Add log snippets and status message to failed/errored builds in overview

### PR DESCRIPTION
**Pull request by TheRealJon**
Add log snippets and status message to failed/errored builds in overview

image

@openshift/team-ux-review

### CODE REVIEW

**Comment by beanh66** [comment]
@TheRealJon I think this looks good :+1:

**Comment by bmignano** [comment]
Looks good to me @TheRealJon !

File: frontend/public/components/overview/build-overview.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      return 'failed';
+    default:
+      return `is ${_.toLower(phase)}`;
+  }
+};
+
+const BuildStatus = ({build}) => {
+  const {status:{logSnippet, message, phase}} = build;
+
+  if (phase !== BUILD_PHASE.Failed) {
```
[the remark this answers is not in the document]
Comment (TheRealJon) [reply frontend/public/components/overview/build-overview.tsx]: This made me find another bug,I wasn't accounting for the Error phase in the BuildPhaseIcon component!

File: frontend/public/components/overview/_build-overview.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  flex-direction: column;
+}
+
+.build-overview__item-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.build-overview__item-reason {
+  border-left: 2px solid $color-pf-red-100;
```
Comment (spadgett) [inline frontend/public/components/overview/_build-overview.scss]: I'd suggest using the same variable for this failed color and pipelines. They're currently the same, but we should use a common var so they always match.

https://github.com/openshift/console/blob/master/frontend/public/components/_build-pipeline.scss#L3

**Comment by TheRealJon** [comment]
@spadgett I've addressed your feedback. Thanks for the review!

**Comment by spadgett** [comment]
@TheRealJon fyi, needs rebase

**Pull request merged**